### ISSUE TITLE
Principal Component Analysis

### ISSUE THREAD

**Issue by douglaszickuhr** [douglaszickuhr/Final_Project#2]
It is needed to deal with PCA in order to scale the data.
This is needed for the future Clustering process.

**Comment by douglaszickuhr** [comment]
Principal component analysis are going to be applied to dimension reduction of variables. 
```{r}
companies_pca <- prcomp(companies_scalled, 
                        scale. = FALSE)
```

**Comment by douglaszickuhr** [comment]
Scales are going to be applied due to difference on the measures of each variable:

1. Total solved complaint - Mean: 695.8
2. Percentage of Solved complaints - Mean: 0.59634 
3. Average weeks to solve a complaint - Mean: 33.03

```{r}
companies_scalled <- companies %>%
  dplyr::select(-c("company_name","company_id")) %>%
  scale(center = TRUE,
        scale = TRUE)

```
